docs/csharp/programming-guide/main-and-command-args/top-level-statements.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Top-level statements - C# Programming Guide"
```

Comment:
The feature is also known as "simple programs" if I recall correctly. You may want to include this name for SEO purposes.

Reply:
Where did you see this name?

Reply:
The public messaging has been consistently to call this feature "top-level statements," and with "simple programs" showing up only in the Roslyn repo, I think using that as an alternate feature name would be unnecessarily confusing, but I'll follow the lead of the top-level statements tutorial and work in the phrase "simple programs" into the text.

Reply:
"Top level statements" is the official name. The product team followed our lead here to avoid the use of "simple". The goal is that programs that use "top level statements" need not be "simple".  See [FeatherApp](https://github.com/featherhttp) for one example of possibilities.